.github/workflows/release.yaml

```diff
@@ -0,0 +1,46 @@
+name: Release
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  build:
+    name: Build and release SemTK
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - shell: bash
+        name: Build SemTK
+        run: |
+          . .env
+          # TODO: The tests currently fail in Github Actions.
+          mvn --quiet --batch-mode clean install -DskipTests
```

Comment:
Another thought about this build step.  We aren't caching ~/.m2 so Maven has to download a bunch of jars every time we run this build step.  If there wasn't a --quiet in the command, we would see the "Downloading" lines in the CI output and realize that we need a cache.  I'd remove the --quiet (the Maven output might be useful to look at sometime) and add a cache step before the build step like:

```
    - name: Cache Maven repository
      uses: actions/cache@v2
      with:
        path: ~/.m2
        key: maven-${{ hashFiles('**/pom.xml') }}
```

Reply:
> I'd remove the --quiet (the Maven output might be useful to look at sometime) and add a cache step before the build step like:

Will do. 

> I'd remove the --quiet (the Maven output might be useful to look at sometime)

I go back and forth on verbosity in CI logs: Often it's unhelpful and results in a lot of scrolling and grepping when a problem arises. On the other hand, as you say, it can be ocassionally useful. And Github Actions at least separates the logs between different steps, which is helpful.